app/code/Magento/InventoryCatalog/view/adminhtml/web/template/components/cell-status.html

```diff
@@ -0,0 +1,13 @@
+<!--
```

Comment:
Pls, provide more clear location for this file according to another `app/code/Magento/InventoryCatalog/view/adminhtml/web/template/product/*`

Reply:
Moved to template/product/form